### DISCUSSION TITLE
Useful Repo to Port over Plugins

### DISCUSSION THREAD

**Discussion by PlatFlat** [Equicord/Equicord#9] [Ideas]
@Equicord I want to express my gratitude for creating this project. It truly enhances the support for third-party plugins.

Here's few repos that could potentially be ported into Equicord.

- https://github.com/cheesesamwich/Tobleronecord/tree/main/src/tobleroneplugins
- https://github.com/fizzexual/FakeDeafenVencord

EDIT: I just noticed FakeDeafen wasn't added because it was against your code of conduct. Although I would have preferred to see it added back, I understand and accept the decision. I had high hopes that Equicord would be more open to allowing plugins like this, otherwise it would be operating in a similar manner to Vencord.

**Comment by KrystalSkullOfficial** [comment]
FakeDeafen is against code of conduct and wont be ever added. Alot of Samwich's plugins are already in Equi
